snmp/files/snmpd.conf

```diff
@@ -177,6 +177,17 @@ rocommunity     {{ rocommunity }}       {{ source }}
 {%- endif %}
 {%- endfor %}
 
+{%- for rocommunity in salt['pillar.get']('snmp:conf:rocommunities6', {}) %}
+{%- set source = salt['pillar.get']('snmp:conf:rocommunities6:'+ rocommunity +':source', '') %}
+{%- if source.__class__ in (().__class__, [].__class__) %}
```

Comment:
Would it be possible to use http://jinja.pocoo.org/docs/dev/templates/#iterable here?